docs/_plugins/include_example.rb

```diff
@@ -38,7 +38,15 @@ def render(context)
       code = File.open(@file).read.encode("UTF-8")
       code = select_lines(code)
  
-      Pygments.highlight(code, :lexer => @lang)
+      rendered_code = Pygments.highlight(code, :lexer => @lang)
+
+      spark_version = site.config['SPARK_VERSION_SHORT']
+      hint = "<div class=\"hint\">Find the example here: " \
```

Comment:
* `Find the example here:` -> `Find full example code here:`
* We don't need a special CSS style to change font size. Use `<div><small>...</small></div>`.